src/sonic-config-engine/openconfig_acl.py

```diff
@@ -18,7 +18,9 @@
 from pyangbind.lib.base import PybindBase
 from decimal import Decimal
 from bitarray import bitarray
-import __builtin__
+
+import builtins
```

Comment:
Do you mind reorganizing the imports while you're at it, per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports)?

>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.


It should be:

```
import builtins
from decimal import Decimal
from operator import attrgetter

from bitarray import bitarray
from pyangbind.lib.yangtypes import RestrictedPrecisionDecimalType, RestrictedClassType, TypedListType
from pyangbind.lib.yangtypes import YANGBool, YANGListType, YANGDynClass, ReferenceType
from pyangbind.lib.base import PybindBase
```

Reply:
done